devices/Ft6xx6x/samples/Program.cs

```diff
@@ -1,55 +1,226 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-
-using System.Device.Gpio;
-using System.Device.I2c;
+using System;
 using System.Diagnostics;
 using System.Threading;
-using Iot.Device.Ft6xx6x;
 using nanoFramework.M5Stack;
+using nanoFramework.M5Core2;
+using Iot.Device.Button;
+using Iot.Device.Ft6xx6x;
+using System.Device.Gpio;
+using System.Device.I2c;
+using System.IO.Ports;
+using UnitsNet;
+using nanoFramework.Runtime.Events;
 
-// Note: this sample requires a M5Core2.
-// If you want to use another device, just remove all the related nugets
-// And comments the following line
-// You will need as well to set the pins if needed.
-M5Core2.InitializeScreen();
-
-I2cConnectionSettings settings = new(1, Ft6xx6x.DefaultI2cAddress);
-using I2cDevice device = I2cDevice.Create(settings);
-using GpioController gpio = new();
-
-using Ft6xx6x sensor = new(device);
-var ver = sensor.GetVersion();
-Debug.WriteLine($"version: {ver}");
-sensor.SetInterruptMode(false);
-Debug.WriteLine($"Period active: {sensor.PeriodActive}");
-Debug.WriteLine($"Period active in monitor mode: {sensor.MonitorModePeriodActive}");
-Debug.WriteLine($"Time to enter monitor: {sensor.MonitorModeDelaySeconds} seconds");
-Debug.WriteLine($"Monitor mode: {sensor.MonitorModeEnabled}");
-Debug.WriteLine($"Proximity sensing: {sensor.ProximitySensingEnabled}");
-
-gpio.OpenPin(39, PinMode.Input);
-// This will enable an event on GPIO39 on falling edge when the screen if touched
-gpio.RegisterCallbackForPinValueChangedEvent(39, PinEventTypes.Falling, TouchInterrupCallback);
-
-while (true)
+namespace NFAppCore2App2
 {
-    Thread.Sleep(20);
-}
+    public class Program
+    {
+        public static void Main()
+        {
+            // Note: this sample requires a M5Core2.
+            // If you want to use another device, just remove all the related nugets
+            // And comments the following line
+            // You will need as well to set the pins if needed.
+            M5Core2.InitializeScreen();
+            nanoFramework.Console.Clear();
+            nanoFramework.Console.WriteLine("Core2 Touch screen test");
+            nanoFramework.Console.WriteLine("Touch Chip type: " + Iot.Device.Ft6xx6x.ChipType.Ft6336U.ToString());
```

Comment:
That is not really useful. You should first read it from the sensor and then display it. So if someone has one from a different family, it will display the right one.

Reply:
What exactly is not useful, the code that appears above your comment is your own code https://github.com/nanoframework/nanoFramework.IoT.Device/blob/develop/devices/Ft6xx6x/samples/Program.cs ???

Reply:
If you are using Visual Studio 2022 or 2019 (the free community version is enough), it will show you the unused ones. The color will be different. On my phone so hard to show you!